Header header
int16 count #当前消息所包含检测目标数量
BoundingBox[] bounding_boxes